airflow/www_rbac/views.py

```diff
@@ -1021,8 +1023,14 @@ def trigger(self, session=None):
             "it should start any moment now.".format(dag_id))
         return redirect(origin)
 
+    @provide_session
     def _clear_dag_tis(self, dag, start_date, end_date, origin,
-                       recursive=False, confirmed=False, only_failed=False, cancel=False):
+                       event_name=None,
```

Comment:
minor nit - can we move the event_name to the last kwarg before session so that the "core" args/kwargs come first? imo makes it more readable.